BackofficeBundle/Resources/public/coffee/tinyMCE/activateTinyMCE.coffee

```diff
@@ -1,7 +1,22 @@
 # ACTIVATE TINYMCE
+
 callback_tinymce_init = null
 
+doCallBack = (editor, view) ->
```

Comment:
@nicolasThal : I dind't introduce that lambda function, I've just updated it. It's a legacy from a preview work of @itkg-rboessel and @itkg-nanne, so they are the best placed to answer this.
But what I have understood discussing with @itkg-nanne is that this mechanism is introduced to deal with a specific Orchestra combo: only one media modal for several rich textes on the same form. That way, each rich text can locally run that callback, and so the modal correctly knows its parent.